MAINTAINING.md

```diff
@@ -100,17 +111,17 @@ that you start from that tag.
 
 To live patch the documentation in between release follow these steps
 
-0. Find the latest documentation release.
+1. Find the latest documentation release.
   - Check the latest release tag (lets say `v0.22.1`).
   - Look for a docs-release tag for that version ex: `v0.22.1-docs.X`
   - If one exists, check-it-out. If not checkout the latest release tag.
   - *Note: Checkout the tag and not master directly because some live
    documentation changes on master that could related to new components
    or updates for the upcoming release*
-0. Create a new branch from there (for example `git checkout -b docs/v0.22.1`)
-0. Cherry-pick the commits you want to include in the live update
+2. Create a new branch from there (for example `git checkout -b docs/v0.22.1`)
+3. Cherry-pick the commits you want to include in the live update
 `git cherry-pick <commit-ish>...`
-0. Use the release-docs script to push and tag to the documentation repository.
+4. Use the release-docs script to push and tag to the documentation repository.
```

Comment:
My vim plugin that fires up a rendered copy while I a make changes was not showing the auto number so I presumed it was not working, looking at github rendering it does work. So, it's a mistake on my part.